slackbridge/bots.py

```diff
@@ -108,4 +111,17 @@ def signedOn(self):
             self.join('#{}'.format(channel['name']))
 
     def post_to_irc(self, channel, message):
-        self.msg(channel, message)
+        self.msg(channel, self.__format_message(message))
+
+    def __format_message(self, message):
```

Comment:
I'd use a single underscore to signify "private" methods instead of double underscores. Apparently there [is actually a difference](https://stackoverflow.com/a/41762806/1979001) in how python treats them.